d3d9ex/AutoFix.cpp

```diff
@@ -390,19 +390,19 @@ void MainContext::FF13_FixScissorRect()
 
 void MainContext::FF13_NOPIngameFrameRateLimitSetter()
 {
-	PrintLog("NOPing the in-game instruction that sets the frame rate.");
+	spdlog::info("NOPing the in-game instruction that sets the frame rate.");
 	MemPatch::Nop(ff13_set_framerate_ingame_instruction_address, 5);
 }
 
 void MainContext::FF13_SetFrameRateVariables()
 {
 	float* framePacerTargetPtr = *ff13_frame_pacer_ptr;
 	if (framePacerTargetPtr) {
-		PrintLog("Frame pacer target frame rate is at address %x", framePacerTargetPtr);
+		spdlog::info("Frame pacer target frame rate is at address {}", (void*)framePacerTargetPtr);

```

Comment:
fmt requires casting pointers to void* to be not ambiguous, then `{}` will format them correctly